modules/monitoring/files/usr/lib/nagios/plugins/check_aws_rds_memory

```diff
@@ -65,8 +65,8 @@ import boto3
 #################################
 parser = argparse.ArgumentParser()
 parser.add_argument("-r","--region", type=str, choices=['us-east-1','eu-west-1'], help="AWS region to check")
-parser.add_argument("-w","--warning", type=float, help="Value(GB) at which to raise a warning alert (5.0)", default=5.0)
-parser.add_argument("-c","--critical", type=float, help="Value(GB) at which to raise a critical alert (2.0)", default=2.0)
+parser.add_argument("-w","--warning", type=float, help="Value(GB) at which to raise a warning alert (50.0)", default=50.0)
+parser.add_argument("-c","--critical", type=float, help="Value(GB) at which to raise a critical alert (20.0)", default=20.0)
```

Comment:
Change to 1.0GB here, please (see above)

Reply:
Done